src/layout/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from 'react';
+
+import { Tabs as DesignsystemetTabs } from '@digdir/designsystemet-react';
+
+import { useRegisterNodeNavigationHandler } from 'src/features/form/layout/NavigateToNode';
+import { useLanguage } from 'src/features/language/useLanguage';
+import { GenericComponent } from 'src/layout/GenericComponent';
+import type { PropsFromGenericComponent } from 'src/layout';
+
+export const Tabs = ({ node }: PropsFromGenericComponent<'Tabs'>) => {
+  const [activeTab, setActiveTab] = useState<string | undefined>(
+    node.item.defaultTab ?? node.item.tabsInternal.at(0)?.id,
+  );
+
+  useRegisterNodeNavigationHandler((targetNode) => {
+    for (const parent of targetNode.parents() ?? []) {
+      const tabsInternal = parent.item['tabsInternal'];
```

Comment:
I'd rather check that `parent.isType('Tabs') && parent.item.id === node.item.id`.

With this code you're indirectly checking that the parent is a `Tabs` component by relying on some internal state unique to the Tabs component right now - but who knows, in the future other components may also get the `tabsInternal` property (just like Grid rows are re-used in RepeatingGroup configuration). Also, you're not checking that the parent is _this_ Tabs component, just any of them. So if you have two Tabs components on the same page, I suspect this will cause tab-switching in the unrelated one.

Reply:
isType() finnes ikke på parent?

Reply:
Ah, `parent instanceof BaseLayoutNode && parent.isType('Tabs')` burde derimot funke